core/src/main/java/io/ultrabrew/metrics/Histogram.java

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019, Oath Inc.
+// Licensed under the terms of the Apache License 2.0 license. See LICENSE file in Ultrabrew Metrics
+// for terms.
+
+package io.ultrabrew.metrics;
+
+import io.ultrabrew.metrics.util.DistributionBucket;
+
+public class Histogram extends Gauge {
```

Comment:
I wrote this on the first agupta comment I saw, but I question whether this should be modeled this way. Will discuss with you both.